source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -623,10 +627,32 @@ def call_kraken(self, request_type, request, instance, krakens_call):
 
         worker = AsyncWorker(instance, krakens_call, request)
 
+        resp = []
+
+        # Now we compute the direct path with all requested departure mode
+        # their time will be used to initialized our PT calls
```

Comment:
update for comments: "their time will also be used to optimize the computation of NM"